### DISCUSSION TITLE
2023-10-25 Weekly Meeting 130 Notes/Agenda

### DISCUSSION THREAD

**Discussion by indiebio** [omigroup/omigroup#423] [Weekly Meetings]
Weekly Meeting 130 is happening on Wednesday, October 25, 2023 from 9:00 AM PDT (16:00 UTC) - 10:00 AM PDT (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).

We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat.

Who we are and why we're here:
The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal.

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in the 🔈 omi-weekly-meeting chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

Agenda:

1. Roundtable
fun, interesting, random or relevant things people have found this week. Please share in the comments.
Why are you here? What do you want from OMI? How can we help each other realise our passions regarding the metaverse?

2. Working group check ins
- jin: did OMI wanna go 50/50 on producing a pitch video for an open metaverse expo? I estimate ~$250 each?

3. Follow on from last week's meeting
Last week https://github.com/omigroup/omigroup/discussions/422

Demo day 30 October 2023. ... move? Or casual meetup in a chosen world?

coinciding with metaverse museum crowdfunder launch. Date changed to 30 October.
CheddarQueso created a flyer - feedback?

Nominations for OMI chairs open, closing after this meeting.
Nominate yourself or someone else. Elections on 1 November 2023.
More info on roles of Chairs https://hackmd.io/@mrmetaverse/omi-tasks-audit;
also: https://github.com/omigroup/omigroup/discussions/157
and https://github.com/omigroup/omigroup/discussions/165

Weekly game night progress? Still to do - look at what is out there already and join up with that. Make an itenirary. e.g. look at https://devpost.com/hackathons

4. Other

Past meeting: https://hackmd.io/@XR/omigroup/https%3A%2F%2Fhackmd.io%2F%40XR%2Fomi_10-12-23

**Comment by indiebio** [comment]
draft flyer from CheddarQueso: 
![image](https://github.com/omigroup/omigroup/assets/23474368/3e9d26fa-1844-4274-bc73-b16c68736ecf)
 - meta-notes ... need something to reuse for events, common style. Thoughts?

**Comment by indiebio** [comment]
jin proposal: 
![image](https://github.com/omigroup/omigroup/assets/23474368/cbb4b50c-24ed-4769-8b54-916012d92cc9)

**Comment by indiebio** [comment]
Roundtable: show and tell, why are you here?

**Reply by indiebio** [reply]
jin: https://fxtwitter.com/decentraland/status/1716881090747772967
[Explore the future of AI at #AIWorldFair from Oct 25-27th. 🤖](https://twitter.com/decentraland/status/1716881090747772967)

**Reply by indiebio** [reply]
jin added two AI bots:
 @Rick Sanchez and @Albus Dumbledore

**Reply by indiebio** [reply]
Why are you here?


- indiebio: real world inspired game, layered, structured data to help real world people collaborate globally, on local issues. "Clusters of villages"
- mtv69: music in the metaverse
- Mal Burns: broadcast news about the metaverse in general. Keeping a thumb on the pulse
- SoloSolopsist / Jeff: layered entertainment. digital twins of living spaces
- Meticulac / Matthew: virtual world as a place to collaborate in art, ideo but immersive. Robotics, virtual world for simulating mechanical stuff
- Saracen: V-Sekai,  metaverse functionality to godot engine
- Keyframe: understanding the evolution of virtual worlds as the seeds of the metaverse - www.virtualworlds.museum
- aaronfranke: Godot engine, 3D asset interoperability
- jin: metaverse stuff, steward metaverse makers - M3, host and build virtual events, platform agnostic artists and activists. cross-pollinating virtual world info. Interested in the in-between and the bigger-than-us. Archivist (OMI-archive). Active MSF - esp avatar wearables forum
- gZ0gJ5SQjQ: Company I work at have some metaverse product, I am here mostly to keep up to date with the technical side of the development of the metaverse
- CheddarQueso: computer science student and metaverse enthusiast here to help the mission.. I believe in the vision of an open metaverse and happy to be in this space where the good conversation happens.
- jimmy6DOF - industry 4.0 digital twins perspective, enterprise applications, 
- mrmetaverse - co-founder OMI, open metaverse believer, we contribute and draw from the metaverse with our arts and experiences. This is an authentic extension of our selves. Interoperability is critical. Trojan horse :) to bring the good stuff through sales / product management. Adoption is our biggest challenge. If we want the metaverse to be people who are able to work together, it starts with us. OMI represents "the people's metaverse"
- Gary:  junior level tech project consultant.  Passionate about creating metaverse in fashion industry.
- Misanth: onboarding solutions and communication in general, sometimes bots, other times 2d design. As long as it revolves around aesthetics, community management, well being, coordination or skill attestation, I'm all for it 🙂 Curious about coordination and ease of access (easily accessible digital co-working spaces etc) 
- avatarjoy: Joyce Bettencourt, a visual designer and community organizer, virtual worlds/Metaverse/VR since 2005. Founder of the US nonprofit AvaCon.org that brings together users and devs of Metaverse technologies, hosting events, highlighting the work done in them. Currently we are in midst of organzing the OpenSimulator community conference for userdevs of the open source virtual world platform software OpenSimulator https://conference.opensimulator.org/, I also host a weekly nonprofit community of practice community in Second Life called Nonprofit Commons


Rick Sanchez the bot - gluing the metaverse connections together (prototype AI test to help with institutional knowledge)

Please feel free to add your contributions!

**Reply by indiebio** [reply]
from jimmy6DOF: Speaking about standards I stumbled across this new sub org of the IEEE for Metaverse standards and they claim to be in alignment with MSF and OM3A etc they did a presentation on it at AWE Europe this week.
https://youtu.be/x0wwvzTqeM8?si=oN9Wcirmt3YJjDTg&t=779

**Reply by indiebio** [reply]
along with years of github notes, we now have 1 year of solid meeting recording + transcripts from weekly community calls + gltf extensions.
reminder: https://hackmd.io/@XR/omigroup/https%3A%2F%2Fhackmd.io%2F%40XR%2Fomi_10-12-23

**Reply by indiebio** [reply]
SoloSolipsist demo: http://halloween23stor.z20.web.core.windows.net/

**Comment by mrmetaverse** [comment]
# Eggs!
https://x.com/mrmetaverse/status/1715505702783889589?s=20

**Comment by indiebio** [comment]
aaronfranke: The scope of OMI

When speaking to people, it is confusing. Do we limit scope? How can we learn from what works?
Who reads our documentation?

Modular approaches... (@madjin  and @aaronfranke please edit this comment with your notes, it's whoosh over my head)

**Reply by indiebio** [reply]
We need multiple messages for multiple uses/applications

**Reply by indiebio** [reply]
Let's focus on what we have in common. - mrmetaverse. And focus on SHOW, not talk so much

**Reply by indiebio** [reply]
conversation to be taken from Fireflies.ai to explain this important conversation

**Reply by aaronfranke** [reply]
Here are some snippets of the conversation I had with Ashxn:

<img width="1343" alt="Screenshot 2023-10-25 at 12 11 16 PM" src="https://github.com/omigroup/omigroup/assets/1646875/d2aaa9b8-b894-4614-ac37-f2cb868465fa">
<img width="1343" alt="Screenshot 2023-10-25 at 12 11 37 PM" src="https://github.com/omigroup/omigroup/assets/1646875/1ab2ceab-54ae-40bf-9669-6923b1660ee2">

TL;DR: Platform stakeholders are skeptical of OMI, but they are interested in VRM. How can we improve how we communicate what OMI protocols are capable of learning from VRM's success?

**Comment by indiebio** [comment]
There is a process for submitting proposals / specs / things - @mrmetaverse please edit and clarify

**Comment by indiebio** [comment]
SoloSolipsist: http://halloween23stor.z20.web.core.windows.net/
conversation about this demo.

**Reply by indiebio** [reply]
avatarjoy: Sharing the demo deck that @funksoup and I and a few others worked on via a hackathon earlier in the pandemic that was built using A-Frame that has some of that people together in spaces https://www.youtube.com/watch?app=desktop&v=WhBfrlK1v30 .. that might be useufl to look at for points you could make on your use case too

**Comment by rick** [comment]
![zuckelback](https://github.com/omigroup/omigroup/assets/6259/ca58c401-1ba2-4a99-8785-33591f83e739)

**Reply by rick** [reply]
https://github.com/rick/you-rang/issues/1
